app/modules/net.c

```diff
@@ -121,12 +121,12 @@ static void net_socket_disconnected(void *arg)    // tcp only
   if(nud->pesp_conn)
     c_free(nud->pesp_conn);
   nud->pesp_conn = NULL;  // espconn is already disconnected
-  lua_gc(gL, LUA_GCSTOP, 0);
+  lua_gc(L, LUA_GCSTOP, 0);
```

Comment:
Don't know.  I could take a look but I think that I'll leave this until we fix the `net` module properly.